### DISCUSSION TITLE
How do you implement multisteps forecaster

### DISCUSSION THREAD

**Discussion by JimmyHoSF** [timeseriesAI/tsai#441] [General]
Which type of multisteps do you implement?  direct mode or recursive mode? thanks

**Comment by oguiza** [comment]
Direct mode.
In my opinion, they tend to work better than in recursive mode, although others may disagree.
Since this is just a question, I'll move it to Discussions.

**Comment by vrodriguezf** [comment]
Is there any literature on this? I prefer direct mode too but it's just because I think it's more elegant and follows the end2end style of deep learning

**Reply by oguiza** [reply]
> Is there any literature on this? I prefer direct mode too but it's just because I think it's more elegant and follows the end2end style of deep learning

I don't know. I'm sure there are papers evaluating these 2 options. I was talking more from my personal experience.
